phase1-coordinator/src/tests.rs

```diff
@@ -1300,6 +1301,386 @@ fn try_lock_blocked_test() -> anyhow::Result<()> {
+            // Completed two verification tasks of the two contributions.
+            assert_eq!(2, verifier_info.completed_tasks().len());
+            assert!(verifier_info.disposing_tasks().is_empty());
+            assert!(verifier_info.disposed_tasks().is_empty());
+        }
+    }
+
+    {
+        let coordinator_state = match storage.read().unwrap().get(&Locator::CoordinatorState).unwrap() {
+            Object::CoordinatorState(state) => state,
+            _ => panic!("unexpected object type"),
+        };
+
+        let contributor1_info = coordinator
+            .current_contributor_info(&contributor1)
+            .unwrap()
+            .expect("expected contributor1 to have info now that it is a current contributor in round 1");
+
+        // Check the coordinator state matches what is in storage.
+        assert_eq!(
+            &contributor1_info,
+            coordinator_state
+                .current_contributor_info(&contributor1)
+                .unwrap()
+                .expect("expected contributor1 to have info now that it is a current contributor in round 1")
+        );
+
+        assert_eq!(1, contributor1_info.assigned_tasks().len());
+        assert!(contributor1_info.pending_tasks().is_empty());
+        assert_eq!(contributor1_n_tasks - 1, contributor1_info.completed_tasks().len());
+        assert!(contributor1_info.disposing_tasks().is_empty());
+        assert!(contributor1_info.disposed_tasks().is_empty());
+
+        let contributor2_info = coordinator
+            .current_contributor_info(&contributor2)
+            .unwrap()
+            .expect("expected contributor2 to have info now that it is a current contributor in round 1");
+
+        // Check the coordinator state matches what is in storage.
+        assert_eq!(
+            &contributor2_info,
+            coordinator_state
+                .current_contributor_info(&contributor2)
+                .unwrap()
+                .expect("expected contributor2 to have info now that it is a current contributor in round 1")
+        );
+
+        assert_eq!(1, contributor2_info.assigned_tasks().len());
+        assert!(contributor2_info.pending_tasks().is_empty());
+        assert_eq!(contributor2_n_tasks - 1, contributor2_info.completed_tasks().len());
+        assert!(contributor2_info.disposing_tasks().is_empty());
+        assert!(contributor2_info.disposed_tasks().is_empty());
+
+        let verifier_info = coordinator
+            .current_verifier_info(&verifier)
+            .unwrap()
+            .expect("expected verifier to have info now that it is a verifier in round 1");
+
+        // Check the coordinator state matches what is in storage.
+        assert_eq!(
+            &verifier_info,
+            coordinator_state
+                .current_verifier_info(&verifier)
+                .unwrap()
+                .expect("expected verifier to have info now that it is a current verifier in round 1")
+        );
+
```

Comment:
No check here for `assigned_tasks` and currently it is empty at this stage, is that to be expected?

Reply:
You can add all the checks which are relevant to this test. I think it's good to both check all the cases and avoid duplication, so the checks may vary from test to test.

Reply:
Thanks, my question was more, are we expecting that the verifier should have assigned tasks at this stage? I was thinking maybe it should, but I wasn't sure, so I didn't commit the check/assertion.